jenkins/job-configs/jenkins-daily-maintenance.yaml

```diff
@@ -16,6 +16,12 @@
             docker rm -v "${containers_to_remove[@]:+${containers_to_remove[@]}}" || true
             docker rmi $(docker images -q -f 'dangling=true') || true
             docker run -v /var/run/docker.sock:/var/run/docker.sock -v /var/lib/docker:/var/lib/docker --rm martin/docker-cleanup-volumes
+            # Clean up leaked VMs from node e2e tests
+            gcloud -q compute instances list \
```

Comment:
Should we run this command after every node e2e iteration instead? Should we be concerned about `gcloud` commands failing when run once a day?